apps/Gruntfile.js

```diff
@@ -4,17 +4,14 @@ var path = require('path');
 var fs = require('fs');
 var os = require('os');
 var _ = require('lodash');
-var webpackConfig = require('./webpack');
-var offlineWebpackConfig = require('./webpackOffline.config');
+var sass = require('sass');
+
 var envConstants = require('./envConstants');
 var checkEntryPoints = require('./script/checkEntryPoints');
-var {BundleAnalyzerPlugin} = require('webpack-bundle-analyzer');
-var CopyPlugin = require('copy-webpack-plugin');
-var {StatsWriterPlugin} = require('webpack-stats-plugin');
-var UnminifiedWebpackPlugin = require('unminified-webpack-plugin');
-var sass = require('sass');
-var TerserPlugin = require('terser-webpack-plugin');
-var {WebpackManifestPlugin} = require('webpack-manifest-plugin');
+
+const { ALL_APPS, appsEntriesFor } = require('./webpackEntryPoints');
+const { createWebpackConfig } = require('./webpack.config')
+const offlineWebpackConfig = require('./webpackOffline.config');
```

Comment:
Any reason we can't `import` these?

Reply:
Unfortunately `Gruntfile.js` (and `webpack*config.js`) are evaluated by node (=outside the babel-loader, and not with modern browser semantics). Because they use the `.js` extension instead of `.mjs`, node, node loads them as considered scripts rather than modules (import allowed). 

Using an `import` gives us:
```
Loading "Gruntfile.js" tasks...ERROR
>> SyntaxError: Cannot use import statement outside a module
Warning: Task "build" not found. Use --force to continue.
```

I think experimenting with possibly turning them into .mjs would be a good idea for a followup PR.